capacity4more/modules/c4m/features_og/c4m_features_og_manage/c4m_manage_categories.inc

```diff
@@ -0,0 +1,834 @@
+      );
+      $form[$key]['parent'] = array(
+        '#type' => 'hidden',
+        // Yes, default_value on a hidden. It needs to be changeable by the javascript.
+        '#default_value' => $term->parent,
+      );
+      $form[$key]['depth'] = array(
+        '#type' => 'hidden',
+        // Same as above, the depth is modified by javascript, so it's a default_value.
+        '#default_value' => $term->depth,
+      );
+    }
+    $path = $menu_item['path'] == 'manage/categories/types' ? 'types' : '';
+    $edit_path = $menu_item['path'] == 'manage/categories/types' ? 'type' : 'term';
+    $form[$key]['edit'] = array('#type' => 'link', '#title' => t('Edit'), '#href' => 'manage/categories/' . $edit_path . '/' . $term->tid . '/edit', '#options' => array('query' => drupal_get_destination()));
+    $form[$key]['delete'] = array('#type' => 'link', '#title' => t('Delete'), '#href' => 'manage/categories/' . $path, '#options' => array('query' => array('delete' => $term->tid)));
+  }
+
+  $form['#empty_text'] = t('No terms available.');
+
+  if ($vocabulary->hierarchy < 2 && count($tree) > 1) {
+    $form['actions'] = array('#type' => 'actions', '#tree' => FALSE);
+    $form['actions']['submit'] = array(
+      '#type' => 'submit',
+      '#value' => t('Save')
+    );
+    $form_state['redirect'] = array($_GET['q'], (isset($_GET['page']) ? array('query' => array('page' => $_GET['page'])) : array()));
+  }
+
+  if ($menu_item['path'] == 'manage/categories') {
+    $form['actions']['new_category'] = array(
+      '#type' => 'link',
+      '#title' => t('Add new type of category'),
+      '#href' => 'manage/categories/types',
+      '#attributes' => array(
+        'class' => array(
+          'btn btn-default',
+        ),
+      ),
+    );
+  }
+
+  return $form;
+}
+
+/**
+ * @TODO: Add docs.
+ */
+function c4m_features_og_manage_categories_overview_form_validate($form, &$form_state) {
+  // Add validate here.
+}
+
+/**
+ * Form builder for the term delete form.
+ *
+ * @ingroup forms
+ * @see taxonomy_term_confirm_delete_submit()
+ */
+function c4m_features_og_manage_taxonomy_term_confirm_delete($form, &$form_state, $tid) {
+  $term = taxonomy_term_load($tid);
+  // Always provide entity id in the same form key as in the entity edit form.
+  $form['tid'] = array('#type' => 'value', '#value' => $tid);
+
+  $form['#term'] = $term;
+  $form['type'] = array('#type' => 'value', '#value' => 'term');
+  $form['name'] = array('#type' => 'value', '#value' => $term->name);
+  $form['vocabulary_machine_name'] = array('#type' => 'value', '#value' => $term->vocabulary_machine_name);
+  $form['delete'] = array('#type' => 'value', '#value' => TRUE);
+  $menu_item = menu_get_item(current_path());
+  $path = $menu_item['path'] == 'manage/categories/types' ? 'types' : '';
+  return confirm_form($form,
+    t('Are you sure you want to delete the term %title?',
+      array('%title' => $term->name)),
+    'manage/categories/' . $path,
+    '<div class="question">'
```

Comment:
Don't put HTML in here.
Why do you repeat the question again in the description?

You can add extra form elements to the $form array (like markup) to add extra help and fields to the form.